From e5cdd413e084538f8b9bf5e6cad8d3dc98ecc48e Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Thu, 2 Jan 2014 13:13:20 +0000
Subject: Allow adding a version-independent directory to the macro path

This allows us to put /usr/share/groff/tmac at the end of groff's macro
path in Debian.

This uses GNU-make-specific code, so it may not be forwardable in its
current state.  A more upstreamable approach might be to add a --with-*
option to configure.

Forwarded: no
Last-Update: 2018-03-05

Patch-Name: extratmacdirs.patch
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d18c49b8..754a8fe7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -211,7 +211,7 @@ localtmacdir=$(dataprogramdir)/site-tmac
 # directory will be always added.
 # `troffrc' and `troffrc-end' (and `eqnrc') are searched neither in the
 # current nor in the home directory.
-tmacpath=$(systemtmacdir)$(RT_SEP)$(localtmacdir)$(RT_SEP)$(tmacdir)
+tmacpath=$(systemtmacdir)$(RT_SEP)$(localtmacdir)$(RT_SEP)$(tmacdir)$(if $(extratmacdirs),$(RT_SEP)$(extratmacdirs))
 
 # sys_tmac_prefix
 # `sys_tmac_prefix' is prefix (if any) for system macro packages.
